software/gatk4/variantfiltration/meta.yml

```diff
@@ -0,0 +1,61 @@
+name: gatk4_variantfiltration
+description: Filter variants
+keywords:
+  - vcf
+  - filter
+tools:
+  - gatk4:
+      description: |
+        Developed in the Data Sciences Platform at the Broad Institute, the toolkit offers a wide variety of tools
+        with a primary focus on variant discovery and genotyping. Its powerful processing engine
+        and high-performance computing features make it capable of taking on projects of any size.
+      homepage: https://gatk.broadinstitute.org/hc/en-us
+      documentation: https://gatk.broadinstitute.org/hc/en-us/categories/360002369672s
+      doi: 10.1158/1538-7445.AM2017-3590
+params:
+  - outdir:
+      type: string
+      description: |
+        The pipeline's output directory. By default, the module will
+        output files into `$params.outdir/<SOFTWARE>`
+  - publish_dir_mode:
+      type: string
+      description: |
+        Value for the Nextflow `publishDir` mode parameter.
+        Available: symlink, rellink, link, copy, copyNoFollow, move.
+  - enable_conda:
+      type: boolean
+      description: |
+        Run the module with Conda using the software specified
+        via the `conda` directive
+  - singularity_pull_docker_container:
+      type: boolean
+      description: |
+        Instead of directly downloading Singularity images for use with Singularity,
+        force the workflow to pull and convert Docker containers instead.
+input:
+  - meta:
+      type: map
+      description: |
+        Groovy Map containing sample information
+        e.g. [ id:'test']
+  - vcf:
+      type: list
+      description: Input VCF file
+      pattern: "*.{vcf}"
+  - fasta:
+      type: tuple of files
+      description: |
+        Tuple of fasta file (first), sequence dict (second) and fasta index (third)
+      pattern: ["*.fasta", "*.dict", "*.fai"]
```

Comment:
Need to add entries here for `fai` and `dict`! Man we need `nf-core lint`!

Reply:
True - this is actually a good test, let's check whether the linter captures this